packages/flutter/lib/src/material/drawer.dart

```diff
@@ -232,16 +262,18 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
     if (_controller.isDismissed)
       return;
     if (details.velocity.pixelsPerSecond.dx.abs() >= _kMinFlingVelocity) {
-      final double visualVelocity = details.velocity.pixelsPerSecond.dx / _width;
+      double visualVelocity = details.velocity.pixelsPerSecond.dx / _width;
+      if(widget.type == DrawerAlignment.end) {
+        visualVelocity *= -1;
```

Comment:
Same point about using unary `-`

Reply:
Fixed. Apologies, this should've been fixed in earlier commit 😐